src/main/java/org/tdl/vireo/model/repo/impl/SubmissionRepoImpl.java

```diff
@@ -745,6 +772,77 @@ public int compare(SubmissionListColumn svc1, SubmissionListColumn svc2) {
 
                     break;
 
+                case "lastAction.entry":
+                    sqlBuilder = new StringBuilder();
+                    if (!sqlJoinsBuilder.toString().contains("LEFT JOIN action_log al ON al.id=s.last_action_id")) {
+                        sqlBuilder.append("\nLEFT JOIN action_log al ON al.id=s.last_action_id");
+                    }
+
+                    sqlJoinsBuilder.append(sqlBuilder);
+                    sqlCountSelectBuilder.append(sqlBuilder);
+
+                    if (submissionListColumn.getSortOrder() > 0) {
+                        setColumnOrdering(submissionListColumn.getSort(), sqlAliasBuilders, sqlOrderBysBuilder, "al.entry");
+                    }
+
+                    for (String filterString : submissionListColumn.getFilters()) {
+                        sqlBuilder = new StringBuilder();
+
+                        if (filterString == null) {
+                            sqlBuilder.append("al").append(".entry IS NULL");
```

Comment:
why not?
```
sqlBuilder.append("al.entry IS NULL");
```

For all of these such cases here...

Reply:
Done.